progress/__init__.py

```diff
@@ -95,8 +96,8 @@ def __init__(self, *args, **kwargs):
 
     @property
     def eta(self):
-        avg = self.avg
-        return int(ceil(self.remaining / avg)) if avg else 0
+        if not self.index: return 0
+        return int(ceil((self.max - self.index) * self.elapsed/self.index))
```

Comment:
There's no reason I'm not using self.remaining.  The reason I disabled the moving average algorithm is that it gives highly variable ETA's for my use case: loops over 1000s of computations with infrequent API calls.  For this - average ETA works better I've found.